modules/mono/csharp_script.cpp

```diff
@@ -1792,6 +1792,34 @@ bool CSharpInstance::has_method(const StringName &p_method) const {
 			gchandle.get_intptr(), &p_method);
 }
 
+int CSharpInstance::get_method_argument_count(const StringName &p_method, bool *r_is_valid) const {
```

Comment:
Will look at writing an instance bridge method for this like the `has_method` part, will see if needed